core/src/main/scala/com/databricks/labs/remorph/parsers/snowflake/SnowflakeAstBuilder.scala

```diff
@@ -56,31 +56,130 @@ class SnowflakeAstBuilder extends SnowflakeParserBaseVisitor[TreeNode] {
   }
 }
 
-class SnowflakeRelationBuilder extends SnowflakeParserBaseVisitor[Relation] {
-  override def visitObject_ref(ctx: SnowflakeParser.Object_refContext): Relation = {
+class SnowflakeRelationBuilder extends SnowflakeParserBaseVisitor[ir.Relation] {
```

Comment:
i have a hunch that we'll have to merge those all files into one, as one ast builder would need methods from another. But perhaps it's a good idea to put it in a separate file for our sanity